config/main.py

```diff
@@ -2881,55 +2881,79 @@ def warm_restart_enable(ctx, module):
 @click.argument('seconds', metavar='<seconds>', required=True, type=int)
 @click.pass_context
 def warm_restart_neighsyncd_timer(ctx, seconds):
-    db = ctx.obj['db']
-    if seconds not in range(1, 9999):
-        ctx.fail("neighsyncd warm restart timer must be in range 1-9999")
-    db.mod_entry('WARM_RESTART', 'swss', {'neighsyncd_timer': seconds})
+    db = ValidatedConfigDBConnector(ctx.obj['db'])
+    if ADHOC_VALIDATION:
+        if seconds not in range(1, 9999):
```

Comment:
I notice many coverage missing are due to lack of unit test for original code. If this is true, could you at least manually test CLI in DUT? @maipbui has similar experience.

Reply:
Yes, I've manually tested every method on DUT CLI